ci-operator/step-registry/cucushift/ota/postupgrade/cucushift-ota-postupgrade-commands.sh

```diff
@@ -248,6 +259,36 @@ function post-OCP-56083(){
     return 1
 }
 
+function post-OCP-23799(){
```

Comment:
It looks like this branch is not update to date. This case should be removed, right?

Reply:
btw, it looks like `23799` is still enabled in 4.16-4.17 upgrade job.

```
$ grep -r "23799" ci-operator/config
ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ota-multi-stable-4.17-upgrade-from-stable-4.16.yaml:      ENABLE_OTA_TEST: OCP-23799

```

Reply:
oh! good one! no idea why it picked that one during rebase :thinking: haven't received a diff for this line, git just merged that in silently :facepalm:

Reply:
ok I've removed both. thanks for noticing :+1:   
btw 4.16-4.17 upgrade job, was not part of the PR that I reverted, but rather added manually by Jianlin in #53553 indeed something to take into account while reverting in this repo :)

Reply:
@evakhoni do you mind to remove the `ENABLE_OTA_TEST: OCP-23799` introduced in https://github.com/openshift/release/pull/53553 ?